aries-site/src/examples/templates/filtering/FilteringCards.js

```diff
@@ -0,0 +1,224 @@
+    status: 'Offline',
+  },
+  {
+    address:
+      '3404 E Harmony Road, Fort Collins, Colorado, 80528, United States',
+    image:
+      'https://images.unsplash.com/photo-1503424886307-b090341d25d1?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1955&q=80',
+    locationType: 'Office',
+    name: 'Fort Collins, CO',
+    status: 'Online',
+  },
+  {
+    address:
+      '6280 America Center Dr., San Jose, California, 95002, United States',
+    image:
+      'https://images.unsplash.com/photo-1501594907352-04cda38ebc29?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2389&q=80',
+    locationType: 'Customer Center',
+    name: 'WW Corporate Headquarters - San Jose, CA',
+    status: 'Online',
+  },
+  {
+    address: '461 Fifth Avenue, New York, NY, 10017, United States',
+    image:
+      'https://images.unsplash.com/photo-1534430480872-3498386e7856?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80',
+    locationType: 'Customer Center',
+    name: 'New York, NY',
+    status: 'Offline',
+  },
+  {
+    address: `Stroombaan 16, 1181 VX Amstelveen, 
+    Amstelveen, The Netherlands`,
+    image:
+      'https://images.unsplash.com/photo-1584450150050-4b9bdbd51f68?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80',
+    locationType: 'Customer Center',
+    name: 'Amstelveen HQ, NL',
+    status: 'Online',
+  },
+];
+
+const StyledCard = styled(Card)`
+  transition: all 0.3s ease-in-out;
+  :focus,
+  :hover {
+    transform: scale(1.01, 1.01);
+  }
+`;
+
+const defaultFilters = {};
+const defaultLocationType = 'All Location Types';
+const defaultStatus = 'All statuses';
+
+export const FilteringCards = () => {
+  const [data, setData] = useState(allData);
+  const [filtering, setFiltering] = useState(false);
+  const [filters, setFilters] = useState(defaultFilters);
+  const [locationType, setLocationType] = useState(defaultLocationType);
+  const [status, setStatus] = useState(defaultStatus);
+  const size = useContext(ResponsiveContext);
+
+  const resetFilters = () => {
+    setData(allData);
+    setStatus(defaultStatus);
+    setLocationType(defaultLocationType);
+    setFiltering(false);
+    setFilters(defaultFilters);
+  };
+
+  useEffect(() => {
+    if (locationType === defaultLocationType && status === defaultStatus) {
+      resetFilters();
+    } else {
+      setFiltering(true);
+    }
+  }, [locationType, status]);
+
+  const filterData = (array, criteria) => {
+    setFilters(criteria);
+    const filterKeys = Object.keys(criteria);
+    return array.filter(item => {
+      // validates all filter criteria
+      return filterKeys.every(key => {
+        // ignores non-function predicates
+        if (typeof criteria[key] !== 'function') return true;
+        return criteria[key](item[key]);
+      });
+    });
+  };
+
+  return (
+    <Box
+      fill
+      gap="medium"
+      margin="auto"
+      overflow="auto"
```

Comment:
Should overflow be moved from here to the Grid / Cards section?

Reply:
Re-pinging this comment.

Reply:
This needs to stay here because it's wrapping the entirety of the layout (similar to the AppContainer on page layouts). Refactoring this wrapper out might help for readability. Can tackle that in the cleanups?

Reply:
Fine to leave here, more was thinking the Results could use an overflow treatment - related: https://github.com/hpe-design/design-system/pull/1084#discussion_r486546883